vispy/visuals/colorbar.py

```diff
@@ -255,7 +253,7 @@ class ColorBarVisual(CompoundVisual):
     pos : tuple (x, y)
         Position where the colorbar is to be placed with
         respect to the center of the colorbar
-    label_str : str
+    label : str
```

Comment:
I'm wondering if we should allow here `TextVisual` too, as we do below in the `label.setter`?